guacamole/src/main/java/org/apache/guacamole/rest/auth/HashTokenSessionMap.java

```diff
@@ -209,11 +209,7 @@ public GuacamoleSession get(String authToken) {
             return null;
 
         // Update the last access time and return the GuacamoleSession
-        GuacamoleSession session = sessionMap.get(authToken);
-        if (session != null)
-            session.access();
-
-        return session;
+        return sessionMap.get(authToken);
```

Comment:
I'm guessing the comment, here, needs to be updated, as well - while it isn't specifically called out, I'm guessing the point is that we no longer want to actually update the session access time when checking the token, because the session has actually been idle?

Reply:
Yep - good catch.

Reply:
Fixed via rebase.